buildSrc/src/main/java/Dependencies.kt

```diff
@@ -235,16 +235,18 @@ object Deps {
     // For the full IDs of these test dependencies, see:
     //   https://developer.android.com/training/testing/set-up-project#android-test-dependencies
     private const val androidx_test_shared_version = "1.4.0" // this appears to be shared with many deps.
+    private const val androidx_test_junit = "1.1.3"
+    private const val androidx_test_orchestrator = "1.4.1"
     const val androidx_test_core = "androidx.test:core:$androidx_test_shared_version"
     private const val androidx_espresso_version = "3.4.0"
```

Comment:
In the next group of bumps, let's group all these const and put them in one place like ones above. Not sure why they are dispersed around the place here.

Reply:
Yes, I'm planning to do a big cleanup & re-org of this file once the current batch of dependency bump PRs lands. I don't want to create more rebase pain in the mean time.

Reply:
Oh fair point. Yeah, this is not worth it then.